### PR TITLE
Added `CustomModules.API.Extensions` and new `CustomRole::Spawn` overload

### PR DESCRIPTION
**Pull request by SmusiJarviss**

### CODE REVIEW

File: Exiled.CustomModules/API/Features/CustomRoles/CustomRole.cs [hunk 1]
```diff
@@ -402,6 +402,21 @@ public static bool Spawn(Pawn player, string name, bool shouldKeepPosition = fal
             return true;
         }
 
+        /// <summary>
+        /// Spawns the specified player with the custom role identified by the provided name.
```
Comment (NaoUnderscore) [inline Exiled.CustomModules/API/Features/CustomRoles/CustomRole.cs:406]:
```suggestion
        /// Spawns the specified player with the custom role identified by the provided type.
```

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Comment by Misfiy** [comment]
If this is no longer draft, mark it as ready for review

**Comment by ALEXWARELLC** [comment]
Since no further additions/changes have been made to this PR, I'll assume it's dead and reopen it as a new PR.

**Comment by ALEXWARELLC** [comment]
@NaoUnderscore Close PR

**Pull request closed without merging**